apps/api/src/app/change/e2e/promote-changes.e2e.ts

```diff
@@ -5,12 +5,16 @@ import { expect } from 'chai';
 import { CreateNotificationTemplateDto } from '../../notification-template/dto/create-notification-template.dto';
 import { UpdateNotificationTemplateDto } from '../../notification-template/dto/update-notification-template.dto';
 import { MessageTemplateRepository } from '../../../../../../libs/dal/src/repositories/message-template/message-template.repository';
+import { NotificationGroupRepository } from '../../../../../../libs/dal/src/repositories/notification-group/notification-group.repository';
+import { EnvironmentRepository } from '../../../../../../libs/dal/src/repositories/environment/environment.repository';
```

Comment:
```suggestion
import { EnvironmentRepository, NotificationGroupRepository } from '@novu/dal';
```

We should only import from the root package, otherwise it might create some weird folder structures in the output of the dist folder.